### PR TITLE
Update benchmark-to-cli branch from dev branch

### PR DESCRIPTION
**Pull request by wannaphong**
fix test ci error.

### CODE REVIEW

File: tests/test_util.py [hunk 1]
```diff
@@ -361,10 +361,11 @@ def test_thaiword_to_date(self):
...
 
-        self.assertEqual(
-            thaiword_to_date("วันนี้").day + 1,
-            thaiword_to_date("พรุ่งนี้").day,
-        )
+        # it's error if "พรุ่งนี้" is 1 not 32.
+        #self.assertEqual(
+        #    thaiword_to_date("วันนี้").day + 1,
+        #    thaiword_to_date("พรุ่งนี้").day,
+        #)
```
Comment (p16i) [inline tests/test_util.py:368]: why do we comment these lines out?

Comment (wannaphong) [reply tests/test_util.py:368]: > 
> 
> why do we comment these lines out?

It has bug. (If date is 31 and tomorrow is 1, `thaiword_to_date("วันนี้").day + 1` is 32)

Comment (p16i) [reply tests/test_util.py:368]: so, we will fix it later? should we create a issue for that?

Comment (p16i) [reply tests/test_util.py:368]: maybe we can just skip the test. For example, we can do something similar to https://stackoverflow.com/a/16138561.

Comment (wannaphong) [reply tests/test_util.py:368]: > 
> 
> so, we will fix it later? should we create a issue for that?

Yes.

**Pull request merged**